react/features/breakout-rooms/components/web/functions.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Generates a class attribute value.
+ *
+ * @param {Iterable<string>} args - String iterable.
+ * @returns {string} Class attribute value.
+ */
+export const classList = (...args) => args.filter(Boolean).join(' ');
```

Comment:
Please let's not duplicate these. Maybe move them to base/ui/styled-utils.web.js or something like that.

Reply:
Agree. For now I will import these functions from the participants-pane and remove this duplicate. In the future the breakout room components needs to be tighter integrated in the participants-pane component.